api/src/pcapi/connectors/boost.py

```diff
@@ -0,0 +1,161 @@
+import datetime
+import enum
+import logging
+from typing import Any
+
+from pcapi.core.external_bookings.boost.exceptions import BoostAPIException
+from pcapi.core.providers.models import BoostCinemaDetails
+import pcapi.core.providers.repository as providers_repository
+from pcapi.repository import repository
+from pcapi.routes.serialization import BaseModel
+from pcapi.utils import requests
+
+
+logger = logging.getLogger(__name__)
+
+
+class ResourceBoost(enum.Enum):
+    EXAMPLE = "example"
+
+
+LOGIN_ENDPOINT = "api/vendors/login"
+
+
+def build_url(cinema_url: str, resource: ResourceBoost) -> str:
+    resource_url = resource.value
+    return f"{cinema_url}{resource_url}"
+
+
+def login(cinema_details: BoostCinemaDetails, ignore_device: bool = False) -> str:
+    """
+    This will provide a JWT for authenticated calls to Boost API.
+    The token has a duration of 24 hours, and is saved in DB along its expiration date.
+    """
+    auth_payload = {
+        "username": cinema_details.username,
+        "password": cinema_details.password,
+    }
+    url = cinema_details.cinemaUrl + LOGIN_ENDPOINT
+    request_date = datetime.datetime.utcnow()
+    try:
+        if ignore_device:
+            response = requests.post(url=url, json=auth_payload, params={"ignore_device": True})
+        else:
+            response = requests.post(url=url, json=auth_payload)
```

Comment:
Pourquoi pas juste ?
```suggestion
        response = requests.post(url=url, json=auth_payload, params={"ignore_device": ignore_device})
```

Reply:
Parce que la fatigue me faire écrire du code retors 😅